README.md

```diff
@@ -4,8 +4,8 @@ teamsnap/emque-consuming](https://www.codeship.io/projects/83d08620-2023-0132-2b
 # Emque::Consuming
 
 Emque Consuming is a Ruby application framework that includes everything needed
-to create and run services capable of consuming messages from a message broker
-in a Pub/sub architecture.  Messages can be produced with the
+to create and run application capable of consuming messages from a message
```

Comment:
I think application gets that across, in that things like passenger are
applications that you can also daemonize. Service, to me, doesn't imply
exactly the same thing, and the layout makes it more difficult to get up to
speed with. That being said, I can be convinced otherwise, but then we
should embrace the name “service” and replace things like
“config/application.rb” with “config/service.rb"

-shane
On Feb 17, 2015 11:26 AM, "Ryan Williams" <notifications@github.com> wrote:

> In README.md
> <https://github.com/teamsnap/emque-consuming/pull/28#discussion_r24828305>
> :
>
> > @@ -4,8 +4,8 @@ teamsnap/emque-consuming](https://www.codeship.io/projects/83d08620-2023-0132-2b
> >  # Emque::Consuming
> >
> >  Emque Consuming is a Ruby application framework that includes everything needed
> > -to create and run services capable of consuming messages from a message broker
> > -in a Pub/sub architecture.  Messages can be produced with the
> > +to create and run application capable of consuming messages from a message
>
> Part of emque-consuming's value proposition (along with the routing) is
> that it can run as a daemon/process and parallelize requests (things you
> can't do with bunny or hutch by themselves). Between service or
> application, or perhaps it just needs to be called out better, which points
> in that direction better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/teamsnap/emque-consuming/pull/28/files#r24828305>.
>